qtpy/tests/test_qtquick.py

```diff
@@ -32,11 +31,11 @@ def test_qtquick():
     assert QtQuick.QQuickTextureFactory is not None
     assert QtQuick.QQuickView is not None
     assert QtQuick.QQuickWindow is not None
-    assert QtQuick.QSGAbstractRenderer is not None
+    # assert QtQuick.QSGAbstractRenderer is not None
     assert QtQuick.QSGBasicGeometryNode is not None
     assert QtQuick.QSGClipNode is not None
     assert QtQuick.QSGDynamicTexture is not None
-    assert QtQuick.QSGEngine is not None
+    # assert QtQuick.QSGEngine is not None
```

Comment:
I think it would be better to add here too a validation to test these modules instead of just commenting the asserts, something like:

```python
if QT5:    # Only available for Qt5 bindings
    assert QtQuick.QSGAbstractRenderer is not None
    assert QtQuick.QSGEngine is not None
```

Reply:
ok